protos/common.proto

```diff
@@ -47,6 +53,13 @@ message Scenery {
 }
 
 message Airbase {
+	string id = 1;
```

Comment:
This doesn't actually work yet. The rust serializer errors out expecting that Id *must* be present

`021-04-02 16:09:24.797 ERROR   SCRIPTING: [GRPC] Error retrieving next command: Failed to deserialize result for method getAirbases: deserialize error: missing field id`

 which seems to be a bug based on https://github.com/danburkert/prost/issues/430 and the comment

> As the README notes, proto3 message fields without a modifier are be wrapped in an Option. The new proto3 optional modifier is for scalar fields, I'm kind of surprised protoc even lets you apply it to a message field.

As a test I did try to apply `optional` to this field but, as expected, this fails.

`protoc failed: common.proto:56:18: Explicit \'optional\' labels are disallowed in the Proto3 syntax. To define \'optional\' fields in Proto3, simply remove the \'optional\' label, as fields are \'optional\' by default`